src/drug_discovery/docking.py

```diff
@@ -38,25 +37,11 @@ def __init__(self, parent):
     def _name_job(self, job: Job) -> str:
         """Generate a name for a job."""
 
-        num_ligands = sum(len(inputs["smiles_list"]) for inputs in job._inputs)
-        return f"Docking protein: <code>{job._metadata[0]['protein_id']}</code> to {num_ligands} ligands."
-
-    def get_results(self) -> pd.DataFrame:
-        """Get docking results from Deep Origin"""
-
-        df1 = self.parent.get_csv_results_for("Docking")
-
-        df2 = ligands_to_dataframe(self.parent.ligands)
-        df2["SMILES"] = df2["Ligand"]
-        df2.drop(columns=["Ligand"], inplace=True)
-
-        df = pd.merge(
-            df1,
-            df2,
-            on="SMILES",
-            how="inner",
-        )
-        return df
+        unique_smiles = set()
+        for inputs in job._inputs:
+            unique_smiles.update(inputs["smiles_list"])
+        num_ligands = len(unique_smiles)
+        return f"Docking <code>{job._metadata[0]['protein_id']}</code> to {num_ligands} ligands."
```

Comment:
Does it matter if its not unique?

Reply:
this is for the case where you dock the same set of ligands many times